### PR TITLE
Cleaned up evoObjects and viewObjects

### LINKED ISSUE #373: Review and clean up class migration files
Clean up analysisObjects.js, modelObjects.js, viewObjects.js, and evoObjects.js, and continue converting classes to backbone

### PR DESCRIPTION
**Pull request by Vbeany**
Closes #373

### CODE REVIEW

File: leaf-ui/js/object/viewObjects.js [hunk 1]
```diff
@@ -23,7 +21,7 @@ class ChartObj {
 					ticks: {
 						min: -2.1,
 						max: 2.1,
-						callback: function (value, index, values) {
```
Comment (krhablutzel) [inline leaf-ui/js/object/viewObjects.js:26]: Did we double-check that these aren't being passed in from elsewhere?

**Review by krhablutzel — APPROVED** [review]
These changes all look great! I had one question about some parameters that got removed, but everything seems to still work well. Thanks for cleaning up these files!

File: leaf-ui/js/object/evoObjects.js [hunk 2]
```diff
@@ -20,37 +19,39 @@ class IntentionColoring {
...
-        this.evals; //list of percentages that each evaluation holds
-        this.timePoints = []; //array of evals at each time point
+        this.evals; // List of percentages that each evaluation holds
+        this.timePoints = []; // Array of evals at each time point
         this.initializeEvalDict();
     }
 
     initializeEvalDict() {
-        this.evals = {};
+        this.evals = {};
```
Comment (Domily11) [inline leaf-ui/js/object/evoObjects.js]: Remove blank

**Review by Domily11 — CHANGES_REQUESTED** [review]
Idk why the intentionColoring class is back but can you remove that and integrate colorblind function into EVO?

File: leaf-ui/js/object/evoObjects.js [hunk 3]
```diff
@@ -1,51 +1,8 @@
...
-     * @param {*} isTurningOnColorBlindMode 
-     */
-    static toggleColorBlindMode(isTurningOnColorBlindMode, analysisResult) {
-        IntentionColoring.isColorBlindMode = isTurningOnColorBlindMode;
-        EVO.refresh(analysisResult);
-    }
-
-}
-
 class IntentionColorVis {
```
Comment (Domily11) [inline leaf-ui/js/object/evoObjects.js:1]: Do we know if this is necessary and if we can merge into evo as well?

**Pull request merged**